.pre-commit-config.yaml

```diff
@@ -82,6 +82,17 @@ repos:
         require_serial: true
         files: ^llama_stack/templates/.*$|^llama_stack/providers/.*/inference/.*/models\.py$
 
+-   repo: local
+    hooks:
+      - id: openapi-codegen
+        name: API Spec Codegen
+        additional_dependencies:
+          - uv==0.6.2
+        entry: sh -c 'uv run --with ".[dev]" ./docs/openapi_generator/run_openapi_generator.sh > /dev/null 2>&1'
```

Comment:
What's the reason for not invoking `uv` directly?

Reply:
suppressing output

Reply:
pre-commit does not output anything, I just tested and `stdout` is not printing. Did I miss something?

Reply:
I see, thanks!